src/CursorPagination.php

```diff
@@ -61,6 +62,8 @@ class CursorPagination implements Paginator
      */
     private ?string $primaryKey = null;
 
+    private ?Closure $keyDecoderCallback = null;
```

Comment:
doc block here

Reply:
and i would use `callable` rather than `Closure` for added flexibility

Reply:
`callable` is not supported and most probably won't be.

Reply:
❓ What's the doc block for as the property has already a type ?

Reply:
ok, sure. I thought this was a Ankorstore guideline :)